src/components/UI/SearchBar/index.tsx

```diff
@@ -11,6 +11,11 @@ import ContextProvider from 'src/state/ContextProvider'
 import FormControl from '../FormControl'
 import './index.css'
 
+function fixFilter(text: string) {
+  const regex = new RegExp(/([\\|*|+|(|)|[|]|])+/, 'g')
```

Comment:
we should consider `?` too in this regexp

Reply:
ok will add it